cmake/YarpBindingsSource.cmake

```diff
@@ -4,16 +4,20 @@
 
 include(GNUInstallDirs)
 
-file(GLOB examples RELATIVE ${CMAKE_SOURCE_DIR}/bindings bindings/example*.*)
-foreach(f CMakeLists.txt yarp.i README swig-allegro-and-chicken-fix.patch compat.h ${examples})
+# Install main CMakeLists and Swig input file
+foreach(f CMakeLists.txt yarp.i README.md)
```

Comment:
I suggest we move the content of this file in the actual place where it is used

Reply:
@drdanz do you mean moving the file or copy-paste the source?

Reply:
@drdanz I copied the content. I will commit now